src/amo/reducers/reviews.js

```diff
@@ -91,6 +98,7 @@ export const initialState: ReviewsState = {
   byId: {},
   byUserId: {},
   latestUserReview: {},
+  ratingSummary: {},
```

Comment:
I am just thinking if this name would be clearer maybe with the name groupedRatings (just to have consistency with the api.?)... or maybe ratingTotals..?